runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/DataflowPipelineJobTest.java

```diff
@@ -359,21 +375,21 @@ public void testGetAggregatorValuesWithNoMetricUpdatesReturnsEmptyValue()
     String stepName = "s1";
     String fullName = "Foo/Bar/Baz";
     AppliedPTransform<?, ?, ?> appliedTransform =
-        appliedPTransform(fullName, pTransform, TestPipeline.create());
+        appliedPTransform(fullName, pTransform, Pipeline.create(options));
 
     DataflowAggregatorTransforms aggregatorTransforms = new DataflowAggregatorTransforms(
         ImmutableSetMultimap.<Aggregator<?, ?>, PTransform<?, ?>>of(aggregator, pTransform).asMap(),
         ImmutableMap.<AppliedPTransform<?, ?, ?>, String>of(appliedTransform, stepName));
 
     GetMetrics getMetrics = mock(GetMetrics.class);
-    when(mockJobs.getMetrics(PROJECT_ID, JOB_ID)).thenReturn(getMetrics);
+    when(mockJobs.getMetrics(PROJECT_ID, REGION_ID, JOB_ID)).thenReturn(getMetrics);
```

Comment:
More out of curiosity than anything else - do you know if there's a compelling reason to use the mockito matchers here? our styling is inconsistent